### PR TITLE
Add Conflux eSpace Mainnet

### PR DESCRIPTION
**Pull request by unchase**
# Add New Chain <chainId>

Thanks for your pull request to add a new support in Sourcify.

If you haven't done so, please follow the instructions on [how to request chain support](https://docs.sourcify.dev/docs/chain-support/) in docs.

Please check the following items before submitting your pull request for a speedy review.

## Checklist

- [x] The branch is named as `add-chain-<chainId>`.
- [x] I haven't modified the `chains.json` file directly.
- [x] In `sourcify-chains.json` file
  - [x] I've set `supported: true`.
  - [x] I haven't added an `rpc` field but the one in [chains.json](../../src/chains.json) is used (if not, please explain why).
- [x] I've added a test in [chain-tests.js](../../test/chains/chains-test.js) file.
- [x] `test-new-chain` test in Circle CI is passing.

image

### CODE REVIEW

File: services/server/src/sourcify-chains-default.json [hunk 1]
```diff
@@ -1380,5 +1380,16 @@
         "url": "https://sepolia-blockscout.scroll.io"
       }
     }
+  },
+  "1030": {
+    "sourcifyName": "Conflux eSpace",
+    "supported": true,
+    "etherscanApi": {
```
Comment (marcocastignoli) [inline services/server/src/sourcify-chains-default.json:1387]: I don't think this explorer is etherscan

**Review by marcocastignoli — CHANGES_REQUESTED** [review]

File: services/server/src/sourcify-chains-default.json (hunk 1 above)
Comment (unchase) [reply services/server/src/sourcify-chains-default.json:1387]: They have a similar API, so they are fully compatible and have passed the necessary CI-tests.

Comment (unchase) [reply services/server/src/sourcify-chains-default.json:1387]: image

Comment (marcocastignoli) [reply services/server/src/sourcify-chains-default.json:1387]: Ok, I was confused by it. I tried it and it works

**Comment by unchase** [comment]
@marcocastignoli conflicts resolved.

File: services/server/src/sourcify-chains-default.json [hunk 2]
```diff
@@ -1381,6 +1381,17 @@
       }
     }
   },
+  "1030": {
+    "sourcifyName": "Conflux eSpace",
+    "supported": true,
+    "etherscanApi": {
+      "apiURL": "https://evmapi.confluxscan.net",
+      "apiKeyEnvName": "CONFLUXSCAN_API_KEY"
```
Comment (kuzdogan) [inline services/server/src/sourcify-chains-default.json:1389]: I believe this explorer does not require a key? Can you confirm? Couldn't find where to get a key

Comment (unchase) [reply services/server/src/sourcify-chains-default.json:1389]: As I understand it, the API key can be obtained as the API replicates the etherscan API and there is mention of different tiers:

image

Comment (kuzdogan) [reply services/server/src/sourcify-chains-default.json:1389]: I couldn't find where to get a key. It seems to work fine without a key anyway

Comment (unchase) [reply services/server/src/sourcify-chains-default.json:1389]: Sure, you can merge it without the key)

**Comment by unchase** [comment]
https://doc.confluxnetwork.org/docs/general/build/tools/web3paywall/

About API keys)

**Comment by kuzdogan** [comment]
Still, I don't know how to get an API key

**Pull request merged**